ledger/internal/eval_blackbox_test.go

```diff
@@ -962,6 +962,57 @@ func benchConsensusRange(b *testing.B, start, stop int, bench func(t *testing.B,
 	}
 }
 
+// TestHeaderAccess tests FirstValidTime and `block` which can access previous
+// block headers.
+func TestHeaderAccess(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	genBalances, addrs, _ := ledgertesting.NewTestGenesis()
+	// Added in v33
+	testConsensusRange(t, 33, 0, func(t *testing.T, ver int) {
+		cv := consensusByNumber[ver]
+		dl := NewDoubleLedger(t, genBalances, cv)
```

Comment:
Is using a double ledger sufficient in showing that 1001 rounds are indeed available in a real ledger?